scripts/client/_otter_macos_env.sh

```diff
@@ -29,7 +29,7 @@ lib_path="${OTTER_SWAC}/lib"
 # its bitmap image format loaders
 export GDK_PIXBUF_MODULE_FILE="$dot_otter_etc/gdk-pixbuf.loaders"
 
-GDK_PIXBUF_MODULEDIR="${lib_path}/gdk-pixbuf-2.0/2.10.0/loaders" \
+GDK_PIXBUF_MODULEDIR="/opt/local/lib/gdk-pixbuf-2.0/2.10.0/loaders" \
```

Comment:
It would be better to leave this as it is and change the declaration of _lib_path_ above instead.